webpack.config.js

```diff
@@ -19,6 +20,12 @@ var define_plugin = new webpack.DefinePlugin({
 });
 
 module.exports = {
+  /* webpack 4
```

Comment:
3 or 4, I would just check on it, not sure if this is intentional or left over code.